Sources/API/Routes/BalanceRoutes.swift

```diff
@@ -47,12 +49,11 @@ public final class BalanceRoutes {
      
      - returns: A StripeRequest<> item which you can then use to convert to the corresponding node
      */
-    public func history(forFilter filter: StripeFilter?=nil) throws -> StripeRequest<BalanceHistoryList> {
+    public func history(forFilter filter: StripeFilter?) throws -> StripeRequest<BalanceHistoryList> {
```

Comment:
I gotcha! Xcode's autocomplete will show you both methods so I think we'd be okay here.